src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/Pledges.cshtml

```diff
@@ -41,7 +59,7 @@
                         {
                             <tr class="govuk-table__row">
                                 <td class="govuk-table__cell">
-                                    <a href="pledges/@pledge.ReferenceNumber/applications" class=govuk-link--no-visited-state>@pledge.ReferenceNumber</a>
+                                    <a href="pledges/@pledge.ReferenceNumber/applications?IsPledgeClosed=@pledge.IsPledgeClosed" class=govuk-link--no-visited-state>@pledge.ReferenceNumber</a>
```

Comment:
See previous comment about this - application state should not be passed around this way.

Reply:
Generic solution to the banner implemented in which TempData is used.